field_char_transformed/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================================
+Transform data on character fields
+==================================
+
+This module was written to give programmers a convenient way to transform text input before it is written into the database or before it is read from it.
+
+As a practical example, FieldCharStripped is provided, which strips whitespace from the value before it is committed.
+
+Configuration
+=============
+
+This module is a dependency to be used by programmers, not end users. There's no configuration to be done via the UI.
+
+Usage
+=====
+
+Depend on this module and import the field classes you need::
+
+    from openerp.addons.field_char_transformed import FieldCharTransformed, FieldCharStripped
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/server-tools/issues/new?body=module:%20field_char_transformed%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Holger Brunn <hbrunn@therp.nl>
+
+Do not contact contributors directly about help with questions or problems concerning this addon, but use the `community mailing list <mailto:community@mail.odoo.com>`_ or the `appropriate specialized mailinglist <https://odoo-community.org/groups>`_ for help, and the bug tracker linked in `Bug Tracker`_ above for technical issues.
```

Comment:
This is not part of the template. I would suggest that you make a PR against https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst if you wish to add a section/text.